freedomofpress/securedrop-proxy
# Add quality control tools

## Description

Added coverage, flake8, mypy, and a few Makefile targets for convenience. Replaced test+safety in CI with just `make check` which runs everything.

This should be dev-only; the change to `requirements.txt` was just to the automatically-generated update instruction comment. I figured we might as well get the new format in place.

## Testing

- run `make` and `make help` to check the target help messages
- try the new targets:
  - `make lint`
  - `make mypy`
  - `make browse-coverage` (particularly if you're on a Mac as I wasn't able to test `open`)
  - `make check`
